dfastmi/gui/dialog_view_model.py

```diff
@@ -105,7 +115,8 @@ def plotting(self):
     @plotting.setter
     def plotting(self, value):
         self._plotting = value
-        self.model.section["Plotting"] = str(value)
+        if self.model and self.model.section :
+            self.model.section["Plotting"] = str(value)
```

Comment:
Would this dict be clashing with the PlotOptions class I introduced in 123? @ralph-peelen-deltares

Reply:
no, but it got me thinking and updated in dialog_model.py